safehttp/form_test.go

```diff
@@ -16,14 +16,13 @@ package safehttp_test
 
 import (
 	"errors"
-	"fmt"
 	"math"
 	"net/http"
 	"net/http/httptest"
-	"reflect"
 	"strconv"
 	"strings"
 	"testing"
+	txttemplate "text/template"
```

Comment:
Do you need the alias?

Reply:
No, I don't, I removed the alias. I followed the example you sent me because I thought it's the preferred practice.

Reply:
Ah, sorry. There was a clash before (I think), or maybe I introduced a clash during development.

In general, we're trying to alias as little as possible. This is also one of the reasons we avoid packages names like `util` or `common`, as people importing them might need to use aliasing a lot.